### DISCUSSION TITLE
[BUG] Auto-completion not work in pwsh

### DISCUSSION THREAD

**Discussion by dysuby** [fastapi/typer#1761] [Questions]
### Describe the bug

I try to install auto completion by running `python main.py --install-completion pwsh` and it executes successfully. But I only get path hint but not command hint.

### To Reproduce

* Create a file `main.py` with:

```Python
import typer


def complete_name():
    return ["Camila", "Carlos", "Sebastian"]


def main(
    name: str = typer.Option(
        "World", help="The name to say hi to.", autocompletion=complete_name
    )
):
    typer.echo(f"Hello {name}")


if __name__ == "__main__":
    typer.run(main)
```

* Run  `python main.py --install-completion pwsh`

### Expected behavior

just help me complete commands

### Screenshots

![image](https://user-images.githubusercontent.com/26317510/114725017-bb57f800-9d6e-11eb-8b5b-3c2d51d888dc.png)

### Environment

* OS: Windows10
* Typer Version: 0.3.2
* python version: 3.9.0

**Comment by dysuby** [comment]
I have run the test case below:
https://github.com/tiangolo/typer/blob/c3a4c72a4f1a18b7f16addb508862ec63411fa63/tests/test_completion/test_completion_install.py#L140-L147

I am not sure this test case can work as we expected, because it write expected output into the profile and assert it is in the profile directly 😂

**Comment by patricksurry** [comment]
I think you must execute the script with `typer` after installing typer-cli,  or install your project as a package, for autocomplete to work.  That way it can customize the autocomplete for your command, rather than generic `python ...`.  See https://typer.tiangolo.com/typer-cli/
